packages/components/src/index.js

```diff
@@ -9,6 +9,7 @@ export { default as CheckboxControl } from './checkbox-control';
 export { default as ClipboardButton } from './clipboard-button';
 export { default as ColorIndicator } from './color-indicator';
 export { default as ColorPalette } from './color-palette';
+export { default as ColorPicker } from './color-picker';
```

Comment:
We are about to add a new component so we should update changelog file and list it as a new feature and propose minor version bump, see: https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs and file to be updated: https://github.com/WordPress/gutenberg/blob/master/packages/components/CHANGELOG.md.